ingest/jobs/Ingest_AzureSql_Example/tests/unit/test_azuresql_example.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from collections import Counter
+from jsonschema import validate
+
+from utils.config_utils import load_config_as_dict, load_configs_as_list
+from utils.constants import INGEST_SOURCES_DIR, INGEST_SOURCES_SCHEMA_PATH
+
+
+def test_config_ingest_sources_schema_valid():
+    schema = load_config_as_dict(INGEST_SOURCES_SCHEMA_PATH, "ingest")
+    all_configs = load_configs_as_list(INGEST_SOURCES_DIR, "ingest")
+    for config in all_configs:
+        validate(config, schema)
+
+
+def test_config_ingest_sources_uniqueness():
+    all_configs = load_configs_as_list(INGEST_SOURCES_DIR, "ingest")
+    assert config_uniqueness_check(all_configs, "data_source_name")
+    for config in all_configs:
+        ingest_entities = config["ingest_entities"]
+        assert config_uniqueness_check(ingest_entities, "display_name")
+
+
+def config_uniqueness_check(config_list: list[dict], unique_key: str):
```

Comment:
same as https://github.com/amido/stacks-azure-data-ingest/pull/36/files#r1196687283 - we will only log the first duplicate.

Reply:
Done